README.md

```diff
@@ -3,7 +3,7 @@
 ![Status of the stable repository](https://github.com/toltec-dev/toltec/workflows/stable/badge.svg)
 ![Status of the testing repository](https://github.com/toltec-dev/toltec/workflows/testing/badge.svg)
 [![rm1](https://img.shields.io/badge/rM1-supported-green)](https://remarkable.com/store/remarkable)
-[![rm2](https://img.shields.io/badge/rM2-experimental-yellow)](https://remarkable.com/store/remarkable-2)
+[![rm2](https://img.shields.io/badge/rM2-supported-green)](https://remarkable.com/store/remarkable-2)
```

Comment:
not sure this is green yet

Reply:
What would it take to be able to say that we fully support rM2, in your opinion?

Reply:
all apps working, bootstrap working, upgrades working (retaining rm2fb shim when toltec re enables) and some community testing period (week or two of it being in stable and people using it?) 

if this is all set, i think it can be green

Reply:
also, would prefer that we take care of oxide and draft service files to launch xochitl if their binaries are unavailable as a failsafe (so one more piece is that sort bricking should be difficult)

Reply:
and uninstaller (if that's not already there), but i put that at p1

Reply:
rM2 apps, bootstrap and upgrades should be working already (if we disregard 2.6 issues, but that is a separate problem).

Having a testing period seems fair.

Launcher failsafe & uninstaller are not available for the rM1 either: should we revert its status back to experimental?

Reply:
unsure, I'm in favor of fixing those, so probably just fix instead of changing badge

Reply:
Just happened to see this convo. I agree with @raisjn's concerns but agree with @matteodelabre's latest comment:

> Having a testing period seems fair.
> 
> Launcher failsafe & uninstaller are not available for the rM1 either: should we revert its status back to experimental?

Requiring features like an uninstaller seems like holding the rM2 badge to a higher standard than the rM1. These features affect toltec in general and not rM2 compatibility.

I can understand the concern about softbricking through failed rm2fb upgrades/re-enables though. This fix would have a higher likelyhood to trigger a soft-brick on rM2 and seems valid for me in that case.

I also think that any bigger update is firstly a stability risk until tested for at least a week by testers. So even with this change being merged into testing, I would still like a more cautious description for compatibility at first. But since this architecture change could also affect the rM 1 compatibility, it shouldn't affect the rM 2 badge status.

Doing ensuring the fallbacks to xochitl in service files, changing the rM 2 bade to "beta" or similar (I would even accept dark green in this case). After a week in either testing or better stable we could update the state in either a merge release or a separate PR.

---

P.S.: I haven't looked much into the changes yet, but overlooking the diff seems promising. Maybe I can give it a try this weekend. Props for that, matteo! :+1: